oak/server/rust/oak_runtime/src/metrics/mod.rs

```diff
@@ -106,6 +108,11 @@ impl GrpcServerMetrics {
                 &["method_name"],
                 "Histogram of response sizes of RPCs handled by the server.",
             )),
+            grpc_server_msg_sent_total: builder.register(histogram_vec(
+                "grpc_server_msg_sent_total",
+                &["method_name"],
+                "Total number of stream messages sent by the server.",
```

Comment:
Maybe `"Total number of gRPC response messages sent by the server."  (I think this will include the counts of responses to unary method invocations, because they're handled as ≈ streaming methods that happen to return a single response, so "stream messages" might be misleading.)

Reply:
Thanks. This is more accurate. I replaced gRPC with RPC to be consistent with the description of the other metrics.